packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -709,8 +709,7 @@
     }
     &__text {
         position: absolute;
-        color: var(--status-success);
+        color: var(--status-success) !important;
```

Comment:
@henry-deriv :) The purpose of the PR is to replace an FE-calculated value with a BE-provided value. why do we need any style changes here? and why are we removing `text-align: center;`?

Reply:
yeah i know but theres a bug on staging at the moment thats why i figured i change here while im at it